lib/src/executable.dart

```diff
@@ -30,6 +34,21 @@ List<String> vmArgsFromArgResults(ArgResults argResults) {
   ];
 }
 
+Future<int> runExecutable(
+    Entrypoint entrypoint, Executable executable, Iterable<String> args,
+    {bool enableAsserts = false,
+    String packagesFile,
+    Future<void> Function(Executable) recompile,
+    List<String> vmArgs = const []}) async {
+  final r = await _runExecutable(entrypoint, executable, args,
+      enableAsserts: enableAsserts,
+      packagesFile: packagesFile,
+      recompile: recompile,
+      vmArgs: vmArgs);
+  print('Returned $r');
+  return r;
+}
```

Comment:
I'm guessing this is a debug hack.. and that `_runExecutable` should replace this.